esp/mlrs-wifi-bt-bridge/mlrs-wifi-bt-bridge.ino

```diff
@@ -213,9 +241,37 @@ void loop()
 
     //-- here comes the core code, handle wifi connection and do the bridge
 
-    uint8_t buf[256]; // working buffer
+    const int bufsize = 256;
+    uint8_t buf[bufsize]; // working buffer
 
-#if WIFI_PROTOCOL == 1 // UDP
+#if (WIFI_nBLUETOOTH == 0) // Bluetooth
+    uint8_t idx = 0;
+    while (SerialBT.available()) {
+      buf[idx++] = SerialBT.read();
+      if (idx >= bufsize) break;
+    }
+    if (idx > 0)
+    {
+      is_connected = true;
+      is_connected_tlast_ms = millis();
+      SERIAL.write(buf, idx);
+    }
```

Comment:
why is this so different than for the UDP case? What am I missing when I think it could be basically similar?

SerialBT.available() returns the number of bytes, pl consider not using while but changing to 
int len = SerialBT.available() 
... ...

it seems SerialBT does not have a read() function usable as SerialBT.read(buf, sizeof(buf)), right?

Reply:
yo, sad. You can mimic it with a simple loop. So I still think that the code could look similar to UDP.

Reply:
Cannot unfortunately follow your reasoning for a change here, as from Bluetooth read one can fish out only byte by byte as long as available returns true and you do want to stop when buffer is filled. Thus a while loop with a break IMO is the optimal solution. No?

Reply:
it might be that one actually has to do if (len > 0) (I think some available() can also return -1)
probably we should to be on the safe side